app/renderer/components/tabs/tab.js

```diff
@@ -324,40 +324,40 @@ class Tab extends ImmutableComponent {
         <div className={css(
           styles.tabId,
           this.narrowView && styles.tabIdNarrowView,
-          this.props.tab.get('breakpoint') === 'smallest' && styles.tabIdMinAllowedSize
+          this.props.frame.get('breakpoint') === 'smallest' && styles.tabIdMinAllowedSize
           )}>
           <Favicon
             isActive={this.props.isActive}
             paintTabs={this.props.paintTabs}
-            tab={this.props.tab}
+            frame={this.props.frame}
```

Comment:
we will remove all this props in redux PR, so I think it's not necessary to do it right now.